projects/ngx-pwa/local-storage/src/lib/storages/storage-map.service.ts

```diff
@@ -162,14 +162,49 @@ export class StorageMap {
    *   }
    * });
    */
+  get<T extends string = string>(key: string, schema: JSONSchemaString): Observable<string | undefined>;
```

Comment:
Would be possible to change this signature (and also `number` signature) to:

```suggestion
  get<T extends string = string>(key: string, schema: JSONSchemaString): Observable<T | undefined>;
```

I'm asking because sometimes we store some pre-defined values, as my real use case when where I store the user selected the preferred theme in settings ("dark", "light"), so I have a type defined as:

`type Theme = 'dark' | 'light'`

... and I would do, with this new signature, as follows:

`.get<Theme>('theme', { type: string })`

... and receive a `Theme` instead of  `string`.

For the `number` signature I also can imagine some specific numbers stored.

Does it makes sense for you?

Reply:
Yes, you're right, thanks for pointing this out! Opened #469, can you check it's OK?